.bit/tests/test.2.7.js

```diff
@@ -33,7 +24,7 @@ if (uri[0] == null) {
         console.log("Yay! 🎉 You're right, you guessed the generation correctly!")
     } else {
         console.log("Try again! We didn't get the correct generation/age back.")
-        console.log(`We got "{$result}" but expected "GenY"`)
+        console.log(`We got "${result}" but expected "GenY"`)
```

Comment:
This could be added to another helper function as in many places we put expected X but got Y, though only benefit would be consistency and you could add the process.exit(1) to that function.

Reply:
I ended up only standardizing placing quotations around each "result" and did not add a helper function since certain outputs require specific comparison statements.